actionview/lib/action_view/helpers/form_helper.rb

```diff
@@ -1682,7 +1683,7 @@ class FormBuilder
       # The methods which wrap a form helper call.
       class_attribute :field_helpers, default: [
         :fields_for, :fields, :label, :text_field, :password_field,
-        :hidden_field, :file_field, :text_area, :checkbox,
+        :hidden_field, :file_field, :textarea, :checkbox,
```

Comment:
shouldn't we leave both textarea and text_area here?

Reply:
I'll take care of it.

Reply:
I followed the precedent established by the `check_box` renaming. 

Do we need entries for both `#check_box` and `#checkbox` as well?

Reply:
No, because `:checkbox` is excluded when defining the helpers.

Reply:
Done in 3278a40d38d8873f238fe3f66211a6c319f3d97c

Reply:
Thank you!

Reply:
Thanks!